cli/README.md

```diff
@@ -184,48 +183,138 @@ mainfluxlabs-cli messages read <channel_id> <thing_auth_token>
 ```
+
 #### Delete group
 ```bash
-mainfluxlabs-cli groups delete <group_id> <user_auth_token>
+mainfluxlabs-cli groups delete <group_id> <user_token>
 ```
-#### Get group with id
+
+#### Get group by id
 ```bash
-mainfluxlabs-cli groups get <group_id> <user_auth_token>
+mainfluxlabs-cli groups get <group_id> <user_token>
 ```
+
 #### List all groups
 ```bash
-mainfluxlabs-cli groups get all <user_auth_token>
+mainfluxlabs-cli groups get all <user_token>
+```
+
+#### Update group
+```bash
+mainfluxlabs-cli groups update '{"id":"<group_id>","name":"newName"}' <user_token>
+```
+
+#### List things by group
+```bash
+mainfluxlabs-cli groups things <group_id> <user_token>
+```
+
+#### View group by thing
+```bash
+mainfluxlabs-cli groups thing <thing_id> <user_token>
+```
+
+#### List channels by group
+```bash
+mainfluxlabs-cli groups channels <group_id> <user_token>
+```
+
+#### View group by channel
+```bash
+mainfluxlabs-cli groups thing <channel_id> <user_token>
+```
+
+### Orgs
+#### Create new org
+```bash
+mainfluxlabs-cli orgs create '{"name":"<org_name>","description":"<description>","metadata":{"key":"value",...}}' <user_token>
+```
+
+#### Get org by id
+```bash
+mainfluxlabs-cli orgs get <org_id> <user_token>
+```
+
+#### List all orgs
+```bash
+mainfluxlabs-cli orgs get all <user_token>
+```
+
+#### Update org
+```bash
+mainfluxlabs-cli orgs update-org '{"id":"<org_id>","name":"new_name"}' <user_token>
+```
+
+#### Delete org
+```bash
+mainfluxlabs-cli orgs delete <org_id> <user_token>
 ```
 
 #### Assign user to an org
 ```bash
-mainfluxlabs-cli orgs assign <user_id> <group_id> <user_auth_token>
+mainfluxlabs-cli orgs assign '{"members": ["member_id":"<member_id>","email":"<email>","role":"<role>"]}' <org_id> <user_token>
 ```
+
 #### Unassign user from org
 ```bash
-mainfluxlabs-cli orgs unassign <user_id> <group_id> <user_auth_token>
+mainfluxlabs-cli orgs unassign '{"member_ids":["<member_id>"]}' <org_id> <user_token>
+```
+
+#### Update member
+```bash
+mainfluxlabs-cli orgs update-member '{"member_id":"<member_id>","org_id":"<org_id>","role":"<new_role>"}' <user_token>
 ```
-#### List users for an org
+
+#### List users by org
 ```bash
-mainfluxlabs-cli orgs members <group_id> <user_auth_token>
+mainfluxlabs-cli orgs members <org_id> <user_token>
 ```
+
 #### List orgs that user belongs to
 ```bash
-mainfluxlabs-cli orgs membership <user_id> <user_auth_token>
+mainfluxlabs-cli orgs memberships <member_id> <user_token>
+```
+
+### Webhooks
+#### Create new webhooks
+```bash
+mainfluxlabs-cli webhooks create '{[{"name":"<webhook_name>","url":"<url>","headers":{"key":"value",...}}]}' <group_id> <user_token>
+```
+
+#### Get webhook by id
+```bash
+mainfluxlabs-cli webhooks get by-id <id> <user_token>
+```
+
+#### Get webhooks by group
+```bash
+mainfluxlabs-cli webhooks get by-group <group_id> <user_token>
+```
+
+#### Update webhook
+```bash
+mainfluxlabs-cli webhooks update '{"id":"<webhook_id>","name":"new_name"}' <user_token>
```

Comment:
Same remark, Is really webhook_id used from the object or from the path? If it's from the path the command should be `mainfluxlabs-cli webhooks update '{"name":"new_name", "url": "myURL"}' <webhook_id> <user_token>`. BTW, if I remember well if you pass this object it should fail because URL is missing, right?